drivers/gles2/rasterizer_storage_gles2.cpp

```diff
@@ -3737,7 +3741,8 @@ void RasterizerStorageGLES2::light_set_param(RID p_light, VS::LightParam p_param
 			light->version++;
 			light->instance_change_notify(true, false);
 		} break;
-		default: {}
+		default: {
```

Comment:
Does your IDE or `clang-format` do that? If the latter, what version do you have? 7.0.1 doesn't modify this code for me.

Reply:
I got that from the clang-format git hook. Looks like I have version 8.0 installed.